### DISCUSSION TITLE
Displaying Validation Loss in mmpretrain Training Process

### DISCUSSION THREAD

**Discussion by yian-zhov** [open-mmlab/mmclassification#1565] [Q&A]
I am able to use mmpretrain for training and each epoch outputs the loss as expected. However, what confuses me is why, during validation, only the accuracy is outputted and there is no loss output for the validation set like during training. Is there a configuration needed in order to achieve this? I haven't found a solution in the official documentation.

**Comment by Ezra-Yu** [comment]
I am sorry that there is no solution to output the loss for the validation set.
